aten/src/ATen/native/cpu/UnaryOpsKernel.cpp

```diff
@@ -15,6 +15,7 @@
 #include <ATen/cpu/vec256/vec256.h>
 #include <ATen/cpu/vml.h>
 #include <ATen/native/Distributions.h>
+#include <ATen/native/Faddeeva.h>
```

Comment:
If we only wanted to include the part of Faddeeva.h we're using into an existing header, like math.h, what would that look like? Would we be able to use it like we do Cephes, with a note for its license and then pointers to that note?

Reply:
I feel it still fits in Math.h. I have moved it there as of now.